karapace/rapu.py

```diff
@@ -18,7 +18,7 @@
 import aiohttp.web
 import aiohttp.web_exceptions
 import asyncio
-import cgi
+import cgi  # pylint: disable=deprecated-module
```

Comment:
I created a separate issue for this: https://github.com/aiven/karapace/issues/533